astrocut/cube_cut.py

```diff
@@ -815,16 +815,23 @@ def cube_cut(self, cube_file, coordinates, cutout_size,
 
         warnings.filterwarnings("ignore", category=wcs.FITSFixedWarning)
         fits_options: Dict[str, Any] = {"mode": "denywrite", "lazy_load_hdus": True}
+
         if cube_file.startswith("s3://"):
             fits_options["use_fsspec"] = True
-            # block size should be:
-            # block_size <= m * hdul[1].section.shape[2] * hdul[1].section.shape[3] * 4 bytes
             fits_options["fsspec_kwargs"] = {"default_block_size": 10_000, "anon": True}
             # only use .section for remote data
             cube_data_prop = "section"
         else:
             fits_options["memmap"] = True
             cube_data_prop = "data"
+
+        # block size should be:
+        # block_size <= m * hdul[1].section.shape[2] * hdul[1].section.shape[3] * 4 bytes
+        if cube_file.startswith("s3://"):
+            with fits.open(cube_file, **fits_options) as cube:
+                block_size = cube[1].section.shape[1] * cube[1].section.shape[2] * cube[1].section.shape[3] * 4
```

Comment:
the m component is dropped here?

Reply:
The m component is `cube[1].section.shape[1]`

Reply:
hm, i thought it would be dependent on the size of the cutout for some reason. 

anyways, this results in a blocksize much larger than what we'd had before (~6 thousand times larger) for the cube `s3://stpubdata/tess/public/mast/tess-s0038-2-2-cube.fits`

I'm seeing: 10,000 vs. 63,311,040

so if this is correct, we should see _MUCH_ fewer requests. And hopefully also about the same amount of bytes downloaded. Are you seeing this?